viewer/packages/data-providers/src/image-360-data-providers/cdm/transformCdmAnnotations.ts

```diff
@@ -70,3 +64,39 @@ export function transformAnnotations(
     } satisfies CoreDmImage360Annotation;
   });
 }
+
+function getEulerRotationOrderFromFormatVersion(formatVersion: string): 'XYZ' | 'XZY' {
+  const semanticVersionNumber = semanticVersionToNumber(formatVersion);
+
+  if (semanticVersionNumber <= 1_000_000) {
+    return 'XYZ';
+  }
+  return 'XZY';
+}
+
+/**
+ * Converts a semantic version string to a comparable number.
+ * @param version The semantic version string (e.g., "1.0.0").
+ * @returns A number representing the semantic version, or 0 if invalid.
+ */
+function semanticVersionToNumber(version: string): number {
```

Comment:
I would personally have preferred an object with `major`, `minor` and `patch` fields, it's much less prone to mistakes, in my opinion